packages/idos-sdk-server-dapp/src/idOS-grantee.ts

```diff
@@ -8,16 +8,17 @@ import * as BorshCodec from "borsh";
 import type { ethers } from "ethers";
 import type { KeyPair } from "near-api-js";
 import nacl from "tweetnacl";
-import { assertNever } from "../types";
-import { EvmGrants, type EvmGrantsOptions, type NearGrantsOptions } from "./grants";
-import type Grant from "./grants/grant";
-import type { GrantChild } from "./grants/grant-child";
-import { KwilWrapper } from "./kwil-wrapper";
-import { implicitAddressFromPublicKey } from "./utils";
 
-/* global crypto */
-
-export type WalletType = "EVM" | "NEAR";
+import {
+  EvmGrants,
+  type EvmGrantsOptions,
+  type NearGrantsOptions,
+} from "../../idos-sdk-js/src/lib/grants";
+import type { GrantChild } from "../../idos-sdk-js/src/lib/grants/grant-child.ts";
+import type Grant from "../../idos-sdk-js/src/lib/grants/grant.ts";
+import { KwilWrapper } from "../../idos-sdk-js/src/lib/kwil-wrapper.ts";
+import { implicitAddressFromPublicKey } from "../../idos-sdk-js/src/lib/utils.ts";
+import { assertNever } from "../../idos-sdk-js/src/types.ts";
```

Comment:
Can we get away with importing from the package base, instead of being married to a FS structure?

Reply:
Hmm, I see your point. Yeah, until we settle the API a bit more, there's not much to gain by having that hard boundary.

Dropping.